src/components/molecules/UserPreferences/Appearance.tsx

```diff
@@ -1,42 +1,52 @@
-import React, { ReactElement } from 'react'
+import React, { ReactElement, ChangeEvent } from 'react'
 import { DarkMode } from 'use-dark-mode'
-import Button from '../../atoms/Button'
 import FormHelp from '../../atoms/Input/Help'
 import Label from '../../atoms/Input/Label'
-import styles from './Appearance.module.css'
 import { ReactComponent as Moon } from '../../../images/moon.svg'
 import { ReactComponent as Sun } from '../../../images/sun.svg'
-
-const buttons = ['Light', 'Dark']
+import BoxSelection, { BoxSelectionOption } from '../FormFields/BoxSelection'
+import Dotdotdot from 'react-dotdotdot'
 
 export default function Appearance({
   darkMode
 }: {
   darkMode: DarkMode
 }): ReactElement {
+  const options: BoxSelectionOption[] = [
+    {
+      name: 'Light',
+      checked: !darkMode.value,
+      title: (
+        <Dotdotdot clamp={1} tagName="span">
```

Comment:
using `react-dotdotdot` and on all the other paces for every content in this new component seems unnecessary. A better way without any of the performance impact of all those truncate components on one screen would be to tweak the space in the boxes. I did that over on another PR for the Appearance boxes, which might be worth trying in here too https://github.com/oceanprotocol/market/pull/552/files#diff-0ec6d11806abee9f7ab3b6fbc4a8353ef37c8af4c1b5d0a01272b19cb2b699be